akka-sample-sharding-scala/killrweather-fog/src/main/scala/sample/killrweather/fog/Fog.scala

```diff
@@ -18,25 +16,27 @@ import com.typesafe.config.Config
 object Fog {
 
   def main(args: Array[String]): Unit = {
-
-    val ports = if (args.isEmpty) Seq(8081) else args.map(_.toInt).toSeq
-
-    ActorSystem[Nothing](Guardian(ports.toVector), "Fog")
+    val weatherApiPorts = if (args.isEmpty) Seq(12553, 12554) else args.map(_.toInt).toSeq
```

Comment:
These are the two seed nodes in config, offset with 1000, so it expects empty args to Fog runs against Killrweather cluster started with empty args